src/main/java/io/odpf/firehose/objectstorage/gcs/GCSObjectStorage.java

```diff
@@ -44,4 +42,16 @@ public void store(String localPath) {
             throw new ObjectStorageFailedException(e);
         }
     }
+
+    @Override
+    public void store(String objectName, byte[] content) throws IOException {
```

Comment:
You can call this method from public void store(String localPath), once you have created objectname from path.

Reply:
ok